### PR TITLE
Moved dev utils to src

### PR DESCRIPTION
**Pull request by maxbarnas**
Resolves: https://github.com/ckeditor/ckeditor5-engine/issues/610.

### CODE REVIEW

File: src/dev-utils/dev-utils.jsdoc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @namespace engine.dev-utils
```
Comment (pjasiun) [inline src/dev-utils/dev-utils.jsdoc]: Can we add here some description what these utils are?

File: src/dev-utils/view.js [hunk 2]
```diff
@@ -3,16 +3,20 @@
...
-import XmlDataProcessor from '/ckeditor5/engine/dataprocessor/xmldataprocessor.js';
-import ViewElement from '/ckeditor5/engine/view/element.js';
-import Selection from '/ckeditor5/engine/view/selection.js';
-import Range from '/ckeditor5/engine/view/range.js';
-import Position from '/ckeditor5/engine/view/position.js';
-import AttributeElement from '/ckeditor5/engine/view/attributeelement.js';
-import ContainerElement from '/ckeditor5/engine/view/containerelement.js';
-import ViewText from '/ckeditor5/engine/view/text.js';
+/**
+ * @namespace engine.dev-utils.view
```
Comment (pjasiun) [inline src/dev-utils/view.js]: Here too.

File: src/dev-utils/model.js [hunk 3]
```diff
@@ -3,33 +3,37 @@
...
-import ViewConversionDispatcher from '/ckeditor5/engine/conversion/viewconversiondispatcher.js';
-import ViewSelection from '/ckeditor5/engine/view/selection.js';
-import ViewDocumentFragment from '/ckeditor5/engine/view/documentfragment.js';
-import ViewElement from '/ckeditor5/engine/view/containerelement.js';
-import ViewAttributeElement from '/ckeditor5/engine/view/attributeelement.js';
-
-import Mapper from '/ckeditor5/engine/conversion/mapper.js';
-import { parse as viewParse, stringify as viewStringify } from '/tests/engine/_utils/view.js';
+/**
+ * @namespace engine.dev-utils.model
```
Comment (pjasiun) [inline src/dev-utils/model.js]: And here.

**Comment by pjasiun** [comment]
You can merge these changes after adding namespace descriptions.

**Comment by maxbarnas** [comment]
I added simple description to namespace and modules. It should be good for now. In future we might remove it (https://github.com/ckeditor/ckeditor5/issues/334) or improve it.

**Pull request merged**